### ISSUE TITLE
dpnp.full() doesn't work with large integer on windows

### ISSUE THREAD

**Issue by chudur-budur** [IntelPython/dpnp#1411]
For example, if we look at the `numpy.full()` implementation, it works:
```python
>>> import numpy
>>> a = numpy.full(3, 4294967295)
>>> a
array([4294967295, 4294967295, 4294967295], dtype=int64)
```

But if we do the same with `dpnp.full()`, it doesn't work:
```python
>>> import dpnp
>>> a = dpnp.full(3, 4294967295)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\user\Softwares\miniconda3\envs\ndpx-win\lib\site-packages\dpnp\dpnp_iface_arraycreation.py", line 739, in full
    return dpnp_container.full(shape,
  File "C:\Users\user\Softwares\miniconda3\envs\ndpx-win\lib\site-packages\dpnp\dpnp_container.py", line 187, in full
    array_obj = dpt.full(shape,
  File "C:\Users\user\Softwares\miniconda3\envs\ndpx-win\lib\site-packages\dpctl\tensor\_ctors.py", line 1105, in full
    hev, _ = ti._full_usm_ndarray(fill_value, res, sycl_queue)
RuntimeError: Unable to cast Python instance to C++ type (#define PYBIND11_DETAILED_ERROR_MESSAGES or compile in debug mode for details)
```
However, `dpnp.full_like()` works:
```python
>>> import dpnp
>>> z = dpnp.zeros(3)
>>> a = dpnp.full_like(z, 4294967295)
>>> a
array([4.2949673e+09, 4.2949673e+09, 4.2949673e+09])
```

Platform and package info:
```python
(ndpx-win) PS C:\Users\user> python
Python 3.8.16 (default, Feb 21 2023, 21:28:58) [MSC v.1929 64 bit (AMD64)] :: Intel Corporation on win32
Type "help", "copyright", "credits" or "license" for more information.
Intel(R) Distribution for Python is brought to you by Intel Corporation.
Please check out: https://software.intel.com/en-us/python-distribution
>>> import platform
>>> platform.platform()
'Windows-10-10.0.19044-SP0'
>>> platform.system()
'Windows'
>>> platform.release()
'10'
>>> platform.version()
'10.0.19044'
>>> import numpy
>>> numpy.__version__
'1.23.5'
>>> import dpnp
>>> dpnp.__version__
'0.11.1'
>>> import dpctl
>>> dpctl.__version__
'0.14.3dev1+115.g18d17281c'
```

**Comment by antonwolfy** [comment]
@chudur-budur , thank you for reporting the issue. We've raised an issue on dpctl [#1230](https://github.com/IntelPython/dpctl/issues/1230).

Just as a workaround for now you can you use the following:
```
a = dpnp.full(3, 4294967295, dtype=dpnp.int64)
```
hope it will not block you much until the issue is resolved in dpct.

**Comment by antonwolfy** [comment]
@chudur-budur, the issue is resolved with dpnp 0.14.0 and dpctl 0.16.0:
```python
>>> import dpnp, dpctl
>>> a = dpnp.full(3, 4294967295)
>>> dpnp.__version__
'0.14.0+189.gfcddad2474'
>>> dpctl.__version__
'0.16.0'
```
